test/integration/etcd/data.go

```diff
@@ -580,7 +580,8 @@ func GetCustomResourceDefinitionData() []*apiextensionsv1beta1.CustomResourceDef
 		// namespaced with legacy version field
 		{
 			ObjectMeta: metav1.ObjectMeta{
-				Name: "foos.cr.bar.com",
+				Name:   "foos.cr.bar.com",
+				Tenant: metav1.TenantSystem,
```

Comment:
why the system tenant here?

Reply:
data.go is the test cases for backward-compatibility, where the tenant is "system". There is another file called multi_tenancy_data.go, where the non-system-tenant test cases are listed.

I added the Tenant here to call it out the difference with the test cases in multi_tenancy_data.go, though in real life the empty Tenant in ObjectMeta will be set by the short-path logic.